test.js

```diff
@@ -2,10 +2,14 @@ import childProcess from 'child_process';
 import test from 'ava';
 import m from './';
 
-test(async t => {
+test('kills a process', async t => {
 	const pid = childProcess.spawn(process.execPath).pid;
 	await m(pid, {force: true});
 
 	// Check if the process exists
 	t.throws(() => process.kill(pid));
 });
+
+test('throws on not found', async t => {
+	await t.throws(m('not-running.exe'));
```

Comment:
You can remove the `async` and `await` keyword. `t.throws` accepts promises as well.